docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -122,6 +122,14 @@
     "hiddenQueries": {
       "type": "boolean"
     },
+    "noConfigPage": {
+      "type": "boolean",
+      "description": "For app plugins that have no config page"
```

Comment:
A little bit confused by this - Does this mean that the app can't be configured at all?

Reply:
just meant they don't have a default config page